hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AbstractAbfsScaleTest.java

```diff
@@ -48,12 +52,23 @@ protected int getTestTimeoutMillis() {
   public void setup() throws Exception {
     super.setup();
     LOG.debug("Scale test operation count = {}", getOperationCount());
-    Configuration rawConfiguration = getRawConfiguration();
+    rawConfiguration = getRawConfiguration();
     assumeScaleTestsEnabled(rawConfiguration);
   }
 
   protected long getOperationCount() {
     return getConfiguration().getLong(AzureTestConstants.KEY_OPERATION_COUNT,
         AzureTestConstants.DEFAULT_OPERATION_COUNT);
   }
+
+
+  /**
+   * Method to get the Huge file for upload value for scale test.
+   * @return the huge value set.
+   */
+  public static int getHugeFileUploadValue() {
```

Comment:
This is never going to pick the value up from the configuration'll inevitably be invoked before the Test suites' setup() is called, so it will be fixed early on.

You may as well be expicit and use new Configuration() in the getTestProperty(); that will have loaded in core-site.xml, its import of auth-keys etc.

+make protected